### PR TITLE
support for 2.13

### PR DESCRIPTION
**Pull request by SwiftEngineer**
Had some time on my hands and wanted to upgrade a personal project that uses this library a lot, so I thought I'd try my hand at solving #38 . At the very least I hope it can be used as a starting point for this project's maintainers.

~Full disclosure: I definitely still don't have a concrete understanding of how this project is being built. I'm also 70% sure that I only ran the JVM and scalaJS tests on 2.13. I would love to help make sure that this whole project is still gravy on 2.12 at least.~

Edit: Can always count on travis to show the way. I'm working on getting those tests passing now.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 sudo: false
 language: scala
+dist: trusty
```
Comment (SwiftEngineer) [inline .travis.yml:3]: Due to: 
https://travis-ci.community/t/expected-feature-release-number-in-range-of-9-to-12-but-got-8-installing-oraclejdk8/1345

File: build.sbt [hunk 2]
```diff
@@ -22,17 +22,22 @@ addCommandAlias("measure-all", "benchmarks/jmh:run -rff results.csv")
...
 addCommandAlias("profile", "benchmarks/jmh:run -prof jmh.extras.JFR -f 1 .*parsebackRun")
 
+publishGithubUser in ThisBuild := "djspiewak"
+publishFullName in ThisBuild := "Daniel Spiewak"
+
 lazy val root = project
   .in(file("."))
   .settings(name := "root")
   .settings(noPublishSettings)
-  .aggregate(benchmarks, coreJVM, coreJS)
```
Comment (SwiftEngineer) [inline build.sbt]: poor benchmarks is being held back by `gll-combinators`. If I get time I'll head there too to help with the push, but for now I just took benchmarks out of the root picture and set it's scalaVersion to be the good ole' fashioned 2.11.

I can see that as being a controversial move, so looking forward to any thoughts on this.

File: core/src/test/scala/parseback/CompoundParserSpec.scala [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
 package parseback
 
-object CompoundParserSpec extends ParsebackSpec {
+class CompoundParserSpec extends ParsebackSpec {
```
Comment (SwiftEngineer) [inline core/src/test/scala/parseback/CompoundParserSpec.scala:19]: Specs weren't running. Switching them from objects to classes seems to have done the trick.

Here's the issue I found that led me to that conclusion: https://github.com/etorreborre/specs2/issues/757

File: core/src/test/scala/parseback/ast/FilterSpecs.scala [hunk 4]
```diff
@@ -171,42 +171,42 @@ object FilterSpecs extends ParsebackSpec {
 
   case class Add(left: Expr, right: Expr) extends Expr with BinaryNode {
     val assocLeft = true
-    val sym = 'add
+    val sym: Symbol = Symbol("add")
```
Comment (SwiftEngineer) [inline core/src/test/scala/parseback/ast/FilterSpecs.scala:174]: Changed due to a deprecation

File: project/build.properties [hunk 5]
```diff
@@ -1 +1 @@
-sbt.version=1.1.1
+sbt.version=1.3.2
```
Comment (SwiftEngineer) [inline project/build.properties:1]: I switched this to use the same version used by the `sbt-spiewak-bintray` plugin.

**Comment by SwiftEngineer** [comment]
Alright. That last commit at least got CI running again. Now it looks like 2.13 is failing due to some deprecations. I'll try and fix those before Monday.

**Comment by djspiewak** [comment]
Very very nice indeed! Thank you! I might know what's going on with the bincompat checks. I'll fiddle with it locally and merge.

**Pull request merged**